types/react-avatar-editor/react-avatar-editor-tests.tsx

```diff
@@ -42,6 +42,6 @@ class AvatarEditorTest extends React.Component {
     }
 }
 
-const getImage: HTMLCanvasElement = AvatarEditor.getImage();
-const getImageScaledToCanvas: HTMLCanvasElement = AvatarEditor.getImageScaledToCanvas();
-const getCroppingRect: CroppedRect = AvatarEditor.getCroppingRect();
+const getImage: HTMLCanvasElement = AvatarEditor.prototype.getImage();
```

Comment:
why not `ref` instead `prototype`? :sweat_smile:

Reply:
Well, I also don't like that... I'll try another way.